CONTRIBUTING.md

```
@@ -159,6 +159,12 @@ final router = GoRouter(
     ),
 ]
 ```
+
+## Regenerating generated files
```

markdownlint-cli2 (0.23.0)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)